index.js

```diff
@@ -82,18 +83,29 @@ function doInject (dispatchFunc, options, callback) {
     }
   }
 
+  const express = options.express
```

Comment:
if instead of defining an express option, we define something with a more generic name?  (in case of another framework that needs it).

Reply:
Feel free to suggest what name you think is more appropriate.

Reply:
No sorry I didn't. 
I don't see how we're disabling request injection here! The aim is to enable request injection "even for express(-like) frameworks".

And since we don't know of any other framework that does the same thing like express (replace req/res proto), I would prefer keeping the flag named as `express`.

Reply:
Good point! I don't know any other framework that does the same thing too. So, express sounds good for now. Maybe in the future, we can change.